### PR TITLE
Show the conversation subtitle on hover

### PR DESCRIPTION
**Pull request by ChristophWurst**
# Before

Bildschirmfoto vom 2021-06-23 09-57-23

# After

Bildschirmfoto vom 2021-06-23 09-56-40

### CODE REVIEW

File: src/components/LeftSidebar/ConversationsList/AppContentListItem/AppContentListItem.vue [hunk 1]
```diff
@@ -112,6 +112,11 @@ export default {
 			type: String,
 			default: '',
 		},
+
+		subtitleText: {
```
Comment (ChristophWurst) [inline src/components/LeftSidebar/ConversationsList/AppContentListItem/AppContentListItem.vue:116]: ideally we could get the text from the slot contents but this turned out to be a non-trivial problem especially with Vue 3

**Comment by nickvergessen** [comment]
I think we need to limit is somehow.

Bildschirmfoto von 2021-06-23 10-29-16

**Comment by nickvergessen** [comment]
Conflicts with #5437

**Comment by ChristophWurst** [comment]
> I think we need to limit is somehow.

Looks good to me actually :)

**Review by marcoambrosini — COMMENTED** [review]
I think that shomething like this should be implemented on the listitem component in the nextcloud vue library

**Comment by skjnldsv** [comment]
> I think we need to limit is somehow.

Make it scroll :D

**Comment by ChristophWurst** [comment]
Guess this one won't make it.

**Pull request closed without merging**